Language/Functions/Bits and Bytes/bit.adoc

```diff
@@ -44,6 +44,32 @@ The value of the bit.
 --
 
 [float]
+=== Example Code
+// Describe what the example code is all about and add relevant code   ►►►►► THIS SECTION IS MANDATORY ◄◄◄◄◄
+Prints the output of `bit(n)` for n=0 to n=10 to the serial monitor. Values returned are 1 2 4 8 16 till 1024.
+[source,arduino]
+----
+void setup() {
+Serial.begin(9600); 
+}
```

Comment:
```suggestion
  while (!Serial) {
    ;  // wait for serial port to connect. Needed for native USB port only
  }
}
```

This will ensure those using a board like the Leonardo will see all the sketch output from n=0.